M2/Macaulay2/packages/NormalToricVarieties/Divisors.m2

```diff
@@ -531,10 +528,7 @@ vertices ToricDivisor := Matrix => D -> (
 latticePoints ToricDivisor := Matrix => D -> (
     V := vertices D;
     if V === null then return null;
-    d := numRows V;
-    V = transpose (normaliz (transpose V,"polytope"))#"gen";
-    s := select (numColumns V, i -> V_(d,i) === 1);
-    c := (V_s)^{0..d-1};
+    c := matrix(vector \ latticePoints convexHull V);
```

Comment:
@ggsmith with this PR, `isVeryAmple` and `latticePoints` for toric divisors are instead using `rawHilbertBasis` (called by Polyhedra after some hoops). Other changes in `NormalToricVarieties` are superficial (e.g. the symbol `isSmooth` is moving to Core, resulting in a bunch of documentation changes).